src/Tabs/index.test.js

```diff
@@ -191,6 +192,7 @@ describe("Tabs", () => {
     [firstTab, secondTab, thirdTab].forEach((tab) => {
       expect(tab.parentElement).not.toHaveAttribute("role", "tab");
       expect(tab.parentElement).not.toHaveAttribute("aria-controls");
+      expect(tab.parentElement).not.toHaveAttribute("aria-selected");
     });
   });
```

Comment:
This assertion is to test that we are NOT applying some ARIA attributes when there are no panels. We should assert `aria-selected` is not present when there are no panels.